vendor/bat-native-ads/src/bat/ads/internal/static_values.h

```diff
@@ -19,28 +19,28 @@ namespace ads {
 
 #define CATALOG_PATH "/v1/catalog"
 
-static const int kIdleThresholdInSeconds = 15;
+const int kIdleThresholdInSeconds = 15;
```

Comment:
Please reverted unrelated changes, i.e. refactor of `static`

Reply:
`const` by default has local linkage. Why do we need `static`?

Reply:
I agree we do not need `static` but this change is unrelated to the issue at this time so should ideally be part of general refactoring

Reply:
I agree with @tmancey because this PR is already really big which makes it hard to review. Better to have more small PRs

Reply:
NONCRITICAL
Covered by: brave/brave-browser#6235 (12)

Reply:
@tmancey I would say this is critical as it just adds noise to this PR

Reply:
@NejcZdovc we consider critical something that breaks functionality. In the follow up issue you will have the chance to revert that change.

Reply:
sure

Reply:
If this broken functionality is merged, anybody who pulls the latest code will have issues until we get another PR merged. Surely best this breaking change is fixed before merging. To me this is critical